cpp/src/arrow/compute/kernels/minmax.h

```diff
@@ -36,11 +36,11 @@ class AggregateFunction;
 /// \class MinMaxOptions
 ///
 /// The user can control the MinMax kernel behavior with this class. By default,
-/// it will return null if there is a null value present.
+/// it will skip null if there is a null value present.
 struct ARROW_EXPORT MinMaxOptions {
-  //   MinMaxOptions() : skip_nulls(false) {}
+  MinMaxOptions() : skip_nulls(true) {}
 
-  //   bool skip_nulls;
+  bool skip_nulls;
```

Comment:
On second thought I think it'd be clearer to have an enumeration like
```suggestion
  enum {
    /// nulls will be ignored
    SKIP,
    /// any nulls will result in null output
    OUTPUT_NULL
  } null_handling;
  bool skip_nulls;
```

Whether or not you choose to use an enum, please explicitly document the return values/validity in both cases

Reply:
@pitrou will this address your concern?

Reply:
It does, but the implementation must be updated for it :-)

Reply:
Though you also need to remove `skip_nulls`.

Reply:
@bkietz @pitrou thanks for the detailed instruction! will change to use enum here